src/main/java/com/gregtechceu/gtceu/common/data/GTMachines.java

```diff
@@ -554,32 +554,37 @@ public class GTMachines {
     //////////////////////////////////////
     // ********* Storage *********//
     //////////////////////////////////////
+
+    public static final BiConsumer<ItemStack, List<Component>> CREATIVE_TOOLTIPS = (stack, components) -> components
+            .add(Component.translatable("gtceu.creative_tooltip.1")
+                    .append(Component.translatable("gtceu.creative_tooltip.2")
+                            .withStyle(style -> style.withColor(GTFormattingCode.RAINBOW.getCurrent())))
```

Comment:
shouldn't this use RAINBOW_SLOW?

Reply:
Creative Data Access Hatch uses RAINBOW, I will standardize them as RAINBOW_SLOW.